public/pages/DetectorDetail/containers/__tests__/DetectorDetail.test.tsx

```diff
@@ -66,7 +66,7 @@ describe('<DetectorDetail /> spec', () => {
   describe('detector detail', () => {
     test('renders detector detail component', () => {
       const { container } = renderWithRouter(detectorId);
-      expect(container).toMatchSnapshot();
+      //expect(container).toMatchSnapshot();
```

Comment:
Local test can pass this UT. Do you know why it fail in Github UT workflow?

Reply:
Github runners run in UTC timezone - when the test runs against the snapshot (which has a converted timestamp time) the test will fail because the converted timestamp in the runner env is different. Ex: tests were failing before here because of a 7-hour offset, which is the difference between PST and UTC

Reply:
Not that this would cause our tests to fail if anyone runs our tests in any timezone other than PST

Reply:
Got it. Then we should skip this first. Let's find way to fix these later. Can you create an issue to track this?

Reply:
What do you mean by skipping? Want me to close PR and create issues for these changes?

Reply:
Not sure if Jest have some annotation to skip some test cases. That's ok we comment out the test case currently. 
No, we can just create a issue like "fix snapshot test with converted timestamp". No need to close this PR.

Reply:
Agreed, yeah. Will create issue. Also yeah, let me change to skipping the test. That way we still preserve the snapshots that can be changed later.